GVRf/Framework/src/org/gearvrf/GVRMesh.java

```diff
@@ -17,20 +17,26 @@
 
 import static org.gearvrf.utility.Assert.*;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import org.gearvrf.utility.Exceptions;
+import org.gearvrf.utility.Log;
 
 /**
  * This is one of the key GVRF classes: It holds GL meshes.
  * 
  * A GL mesh is a net of triangles that define an object's surface geometry.
  */
-public class GVRMesh extends GVRHybridObject {
+public class GVRMesh extends GVRHybridObject implements PrettyPrint {
```

Comment:
Not sure why PrettyPrinting needs to be brought into core classes. There could be an external PrettyPrinting class that knows all it needs to know about certain GVRF classes and keep all the formatting in one place. Now we are polluting core classes with public methods and code that doesn't really belong to them.